### DISCUSSION TITLE
Hope to support Chinese language

### DISCUSSION THREAD

**Discussion by Zxxxxxl** [massCodeIO/massCode#120] [Q&A] [answered]
I especially like this software，Very convenient and fast。But my English is not very good，I hope it can adapt to Chinese language.  very think。

**Comment by antonreshetov** [comment, marked as answer]
I do not know Chinese at all and I'm not very good at English, either 😄 In general, the application interface is very simple, and I think it is intuitive. Maybe in the future I will consider the possibility of internationalization.

**Reply by HeliusHao** [reply]
Can you make the language file in xml file or ini file format, so we can translate the xml and ini cross-reference table by ourselves?

**Reply by AuroraTea** [reply]
Look forward to internationalization and hope I can help in any way I can

**Comment by antonreshetov** [comment]
@HeliusHao @sangyuxiaowu

I just added [i18n](https://github.com/massCodeIO/massCode/pull/152) support. Need your help in translation. See [README](https://github.com/massCodeIO/massCode/tree/master/src/main/services/i18n/locales) to understand how to do it.
